backend/cmd/headlamp.go

```diff
@@ -60,28 +62,34 @@ type HeadlampConfig struct {
 
 const PodAvailabilityCheckTimer = 5 // seconds
 
+const (
+	RUNNING = "Running"
+	STOPPED = "Stopped"
+)
+
 type PortForward struct {
-	ID        string `json:"id"`
-	closeChan chan struct{}
-	Pod       string `json:"pod"`
-	Service   string `json:"service"`
-	Namespace string `json:"namespace"`
-	Cluster   string `json:"cluster"`
-	Port      string `json:"port"`
-	Out       *bytes.Buffer
-	ErrOut    *bytes.Buffer
-	OutString string `json:"outString"`
-	ErrString string `json:"errString"`
+	ID               string `json:"id"`
+	closeChan        chan struct{}
+	Pod              string `json:"pod"`
+	Service          string `json:"service"`
+	ServiceNamespace string `json:"serviceNamespace"`
+	Namespace        string `json:"namespace"`
+	Cluster          string `json:"cluster"`
+	Port             string `json:"port"`
+	TargetPort       string `json:"targetPort"`
+	Status           string `json:"status"`
+	Error            string `json:"error"`
 }
 
 type PortForwardPayload struct {
-	ID         string `json:"id"`
-	Namespace  string `json:"namespace"`
-	Pod        string `json:"pod"`
-	Service    string `json:"service"`
-	TargetPort string `json:"targetPort"`
-	Cluster    string `json:"cluster"`
-	Port       string `json:"port"`
+	ID               string `json:"id"`
+	Namespace        string `json:"namespace"`
+	Pod              string `json:"pod"`
+	Service          string `json:"service"`
+	ServiceNamespace string `json:"serviceNamespace"`
```

Comment:
Instead of having `Namespace`, `Pod`, `Service` and `ServiceNamespace` I think it is better to have a `Namespace`, `Name`  and `Type` (which takes they value `service` or `pod`)

Reply:
We have case like where we have podName, but there is service name too for that this would not be enough

Reply:
So the service namespace can be different than the pod's namespace? I would assume that the service and the pod's name would be the same...

Reply:
I found usecases where a service in a different namespace can control a pod from a different namespace, thus having these two seperate covers all the usecases for the service